modules/vba_documents/app/models/vba_documents/upload_submission.rb

```diff
@@ -94,10 +93,8 @@ def self.refresh_statuses!(submissions)
         end
         ActiveRecord::Base.transaction { updated.each(&:save!) }
       else
-        submissions.first.log_message_to_sentry('Error getting status from Central Mail API',
-                                                :warning,
-                                                status: response.status,
-                                                body: response.body)
+        Rails.logger.warn('Vba_documents API: Error getting status from EMMS API.' \
+                          " EMMS Response Status: #{response.status}. EMMS Response Body: #{response.body}")
```

Comment:
Similar to my comments above, I think we should keep the "Central Mail" wording, and this is another opportunity for the following:
`Rails.logger.warn('Error getting status from Central Mail API.', { central_mail_status: response.status, central_mail_body: response.body })`

Reply:
changed